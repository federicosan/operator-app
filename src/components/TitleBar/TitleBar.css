#title {
  background: #666666;
}

#title span {
  display: block;
  padding: 9px 0;
  text-transform: uppercase;
  text-align: center;
  /*text-shadow: 0 0 2px #aaaaaa;*/
  font-weight: 800;
  font-size: 22px;
  color: white;
}