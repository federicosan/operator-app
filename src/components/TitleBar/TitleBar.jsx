import React from 'react';
import './TitleBar.css';

const TitleBar = () => (
  <div id="title" className="row bar">
    <span>Operator</span>
  </div>
);

export default TitleBar;
