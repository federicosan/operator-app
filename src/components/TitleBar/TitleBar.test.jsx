import React from 'react';

import { shallow } from 'enzyme';

import TitleBar from './TitleBar.jsx';

const store = {
  subscribe: jest.fn(),
  dispatch: jest.fn(),
  getState: jest.fn(() => ({ })),
};

describe('TitleBar', () => {
  it('matches snapshot', () => {
    const component = shallow(<TitleBar store={store} />);

    expect(component).toMatchSnapshot();
  });
});
