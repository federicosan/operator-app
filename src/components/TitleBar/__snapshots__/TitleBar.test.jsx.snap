exports[`TitleBar matches snapshot 1`] = `
<div
  className="row bar"
  id="title">
  <span>
    Operator
  </span>
</div>
`;
